doc/classes/VisualServer.xml

```diff
@@ -1108,6 +1108,65 @@
 				[/codeblock]
 			</description>
 		</method>
+		<method name="fti_instance_create">
+			<return type="RID" />
+			<description>
+				FTI ("fixed timestep interpolation") instance provides a convenience physics interpolation wrapper for users creating instances directly on the server rather than through Nodes.
```

Comment:
```suggestion
				FTI ("fixed timestep interpolation") instances provide convenience physics interpolation wrappers for users creating instances directly on the server rather than through Nodes.
```
Or
```suggestion
				An FTI ("fixed timestep interpolation") instance provides a convenience physics interpolation wrapper for users creating instances directly on the server rather than through Nodes.
```